src/encoded/static/scss/bootstrap/_forms.scss

```diff
@@ -124,7 +124,6 @@ output {
   border: 1px solid $input-border;
   border-radius: $input-border-radius;
   @include box-shadow(inset 0 1px 1px rgba(0,0,0,.075));
-  @include transition(border-color ease-in-out .15s, box-shadow ease-in-out .15s);
```

Comment:
Great catch. I am used to bootstrap being used as a single huge file rather than using component-files; so this slipped past me. I did an approach-reset. I just reset transition to default values only on the input text box of interest and not the entire site.